documentation/developers.md

```diff
@@ -88,10 +89,29 @@ web-router-service: {
   }
 }
 ```
-
 For information on proxying plaintext `/status` requests to an otherwise HTTPS
 protected server, see the [Status Proxy documentation](./status-proxy-service.md).
 
+The status service can interact with the tk-auth service if it is also a
+dependency. If it is a dependency, it is possible to require authentication for
+the status endpoint.  You can do this by updating the
+/etc/puppetlabs/puppetserver/conf.d/auth.conf file to include this authorization rule:
```

Comment:
I'd just say something like "tk-auth's configuration file" or something less specific than the actual path to where Puppet Server puts its tk-auth file (assuming the primary consumers of this documentation are either people outside the Puppet Server team within Puppet or community members using tk-status as a library in their own applications).